modules/cluster-logging-deploy-clo.adoc

```diff
@@ -7,39 +7,45 @@
 
 You can use the {product-title} web console to install the Cluster Logging Operator.  
 
+[NOTE]
+====
+You cannot create a Project starting with `openshift-` using the web console or 
+by using the `oc new-project` command. You must create Namespace using a YAML 
+object file and run the `oc create -f <file-name>.yaml` command, as shown.
+==== 
+
 .Procedure
 
 To install the Cluster Logging Operator using the {product-title} web console:
 
-. Create a Namespace for the Cluster Logging Operator:
-
-.. In the {product-title} web console, click *Home* -> *Projects*.
-
-.. Click *Create Project*.
-
-.. Enter `openshift-logging` in the *Name* and *Display Name* fields.
-
-.. Optional: Enter a description.
-
-.. Click *Create*.
+. Create a Namespace for the Cluster Logging Operator. You must use the CLI to create the Namespace.
 
-.. On the *Project Details* page, click *YAML*.
-
-.. Replace the YAML shown on the page with the following:
+.. Create a Namespace object YAML file (for example, `clo-namespace.yaml`) for the Cluster Logging Operator:
```

Comment:
Maybe avoid the parenthetical and just include the example file name as the title of the following codeblock. I.e.,:

```
.`clo-namespace.yaml` file
[source,yaml]
----
apiVersion: v1
kind: Namespace
```